spec/anthropic/completion_spec.rb

```diff
@@ -49,37 +49,44 @@
   end
 
   context "with successful completion", :success do
-    let(:completion) { anthropic.complete(LLM::Message.new("user", "Hello, world")) }
+    let(:message) { LLM::Message.new("user", "Hello, world") }
+    let(:completion) { anthropic.complete(message) }
+
+    it "returns completion" do
```

Comment:
Small typo. Should be: `returns a completion`

Reply:
Thanks!